index.md

```diff
@@ -4,113 +4,117 @@ layout: essential
+{:.using-cockpit}
+- Diagnose network issues
+- Spot and react to misbehaving virtual machines
+- Examine SELinux logs and fix common violations in a click
+- Inspect detailed metrics that correlate CPU load, memory usage, network activity, and storage performance with the system's journal
 
----
+More features appear in Cockpit every release.
 
-## Glanceable
+### Designed & tested
 
-### System overview
-Immediately understand the health of your server. Cockpit's overview page shows current statistics and the status of your system.
+Cockpit's design keeps your goals in mind.  We test Cockpit with usability studies to make it work the way you'd expect and adjust accordingly. As a result, Cockpit gets easier to use all the time.
 
-### Multi-server
-Monitor and administer [several servers]({{ site.baseurl }}/guide/latest/feature-machines.html) at the same time. Add new hosts and your main server will look after its buddies.
+All code changes have tests which must pass before merging, to ensure stability.
 
-[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png)
-{:.screenshot.zoom}
+### Free & free
 
-### Troubleshoot
-Fix pesky problems with ease.
+Cockpit is free to use and [available under the GNU LGPL](https://github.com/cockpit-project/cockpit/blob/master/COPYING).
 
-- Diagnose network issues
-- Spot and react to misbehaving virtual machines
-- Inspect SELinux logs and fix common violations in a click
+### Cockpit works (nearly) everywhere
 
----
+You can install Cockpit on the major distributions, including:
 
-## Open-ended
+{:.distro-logos}
+{%
+  for distro in site.data.distros
+%}- [![{{ distro.name }}](/images/site/os-{{ distro.first}}.svg)]({{ site.baseurl }}/running.html#{{ distro.first }})
+{% endfor %}
 
-### Pocket-sized
-Around the office or on the road? Use your phone's browser to check and manage systems.
+Once Cockpit is up and running, you can access systems from all major web browsers on any operating system (including Windows, MacOS, and Android).
 
-### Learn with Cockpit
-Not sure what you can do with your servers? Explore Cockpit's friendly interface and use features you might not even realized existed!
+## Release schedule
 
-### Extendable
-Have custom tasks? You can [write your own modules]({{ site.baseurl }}/blog/cockpit-starter-kit.html) to plug into Cockpit.
+Cockpit has a time-based release cadence, with new versions appearing every two weeks.
 
-### Free & free
-Cockpit's completely free to use and [available under the GNU LGPL](https://github.com/cockpit-project/cockpit/blob/master/COPYING).
+## Get started
 
+{:style="text-decoration-skip-ink:none"}
+After [installing and enabling Cockpit]({{ site.baseurl }}/running.html), visit port 9090 on your server (for example: <https://localhost:9090/> in a browser on the same machine as Cockpit).
```

Comment:
For localhost we could even use http://. That would avoid the pesky self-signed certificate problem, and for localhost you don't gain anything with encryption.

Of course that does not translate to remote hosts, but there cockpit automatically redirects http to https anyway. So maybe overall this would be a better choice? This could even format the link to not show the schema at all, like 

    After..., [visit port 9090 on your machine](http://localhost:9090), or on your remote server.

?

Reply:
I had the URL exposed to demonstrate that it would go to one's local machine. If it's hidden away, it's just a broken link for those who aren't using Cockpit.

Generally, I don't like exposing URLs as linked text, but thought it made sense in this case.

We _could_ use a hybrid approach and make the link contain the text `localhost:9090` (not showing `http`/`https` and the `://`), which also happens to work in browsers just fine.

Reply:
Good point about being explicit. Then I think it's not actually bad to show http://localhost:9090 explicitly.